tests/io/test_formats.py

```diff
@@ -19,6 +19,19 @@
 from sleap.gui.app import MainWindow
 from sleap.gui.state import GuiState
 from sleap.info.write_tracking_h5 import get_nodes_as_np_strings
+from sleap.io.format.sleap_analysis import SleapAnalysisAdaptor
+
+
+def test_sleap_analysis_read(single_fly_mp4_vid, single_fly_hdf5):
```

Comment:
I know we needed a single-instance hdf5 file, but do you mind reusing one of the videos we already have? I don't think we really need to add a whole new video to out test data.

Reply:
Yes, I have used a smaller one in my latest commit